mocked-api/handlers.ts

```diff
@@ -1202,69 +1132,31 @@ const prepareProcessor = (
   return omit(processor, omitProperties) as ProcessorResponse;
 };
 
-/**
- * Convert actions/sources parameters to string because that's how they are stored
- * in our fake db
- *
- * @param data Processor action or source to convert
- * @returns Processor action or source with parameters property containing stringified parameters.
- */
-const convertParametersToString = (
-  data: MockProcessorRequest["action"] | MockProcessorRequest["source"]
-): { type: string; parameters: string } => {
-  // converting secrets fields to an empty object to simulate their server side obfuscation
```

Comment:
I've opted for removing this because it's not needed for now, at least for processors.
If we'll need something similar for sources/sink we can restore it from history.